App_Programmering/Produkt_beskrivelse.md

```diff
@@ -0,0 +1,17 @@
+# Produkt beskrivelse
+
+__Hvilken form for app:__
+Jeg kunne godt bare tænke mig at udvilke mig fagligt, i forhold til at prøve lidt af hvert. Kan ikke helt beslutte mig for om jeg vil udvikle i React eller Android studio.
```

Comment:
Hvornår beslutter du dig for hvilken platform du gerne vil udvikle i?

Reply:
Jeg har besluttet mig for at arbejde i Android studio.
Så har jeg også fjernet Database delen.